### PR TITLE
Add a test to validate WDLs in the scripts directory.

### PR DESCRIPTION
**Pull request by gbggrant**

### CODE REVIEW

**Comment by gbggrant** [comment]
@cmnbroad I saw your comment in https://github.com/broadinstitute/gatk/pull/7822. I'm trying to add WDL validation here, using the infrastructure in gradle. I was interested here in just validating the WDLs in the scripts directory.

File: .github/workflows/gatk-tests.yml [hunk 1]
```diff
@@ -251,6 +251,19 @@ jobs:
           identifier: Java ${{ matrix.Java }} test on docker ${{matrix.testType}}
           only-artifact: ${{ needs.check-secrets.outputs.google-credentials != 'true' }}
 
+  #test wdl validation
+  wdlValidation:
+    runs-on: ubuntu-latest
+    name: Validate WDLs using womtools
```
Comment (cmnbroad) [inline .github/workflows/gatk-tests.yml:257]: Can you modify the names (workflow names, name property, gradle task names, and comments) so the "scripts" and "generated" versions are more clearly distinguished from each other (i.e., `validteScriptsWDL` and `validateGeneratedWDL` or something similar). Its hard to tell from looking at just this definition that it's not doing the autogenerated validation.

**Review by cmnbroad — APPROVED** [review]
@gbggrant One comment on naming, otherwise looks good to merge once thats addressed. Thanks for doing this.

**Review by mcovarr — APPROVED** [review]
👍 with Chris's suggestions

**Pull request merged**